modules/odo-debugging-changes.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/developer_cli_odo/odo-migration-guide.adoc
+
+:_content-type: CONCEPT
+[id="odo-debugging-changes_{context}"]
+
+= Changes to component debugging
+
+With `odo` v2, you use the `odo push --debug` command to run a component in debug mode. To set up port forwarding to the component's debug port, you have to run `odo debug port-forward`.
```

Comment:
Maybe I'm overthinking this, but would this content make sense as a small procedure? Something like:

= Changing component debugging 
For both `odo` v2 and v3, you can debug your components. 

== Debugging components in `odo` v2

1. Run a component in debug mode by running, `odo push --debug`.
2. Set up port forwarding to the component's debug port by running, ``odo debug port-forward`.

== Debugging components in `odo` v3

1. Specify the debug port in the `devfile.yaml` as an endpoint.
2.  Run `odo dev --debug`.
3. Start the component in debug mode. 

The following example shows a `container` component in the Devfile, where port `3000` is the application port and `5858` is the debug port: (provide your sample devfile yaml that you have)

Reply:
That would be good, but would require that I have the YAML/config for V2 which is different to what is available for V3. SO I am going to leave it as is, just in case I screw it up with the wrong set of steps